src/GovUk.Education.ExploreEducationStatistics.Data.Model/Database/StatisticsDbContext.cs

```diff
@@ -102,6 +102,7 @@ private void Configure(int? timeout = null, bool updateTimestamps = true)
             }
         }
 
+        public DbSet<BoundaryData> BoundaryData { get; set; } = null!;
```

Comment:
I think you can delete the `GeoJson` type now mentioned in L113 below and the reference to the `geojson` view in method `ConfigureGeoJson`.